index.js

```diff
@@ -1,18 +1,68 @@
 /***** Deliverable 1 *****/
 const header = document.querySelector("#header")
-console.log("Here's your header:", header)
+// console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
-
+header.style.color = 'orange'
 
 
 /***** Deliverable 3 *****/
 // console.log('This is what the traveler object looks like: ', traveler)
-
+const image = document.querySelector('#profile img')
+const h2 = document.querySelector('#profile h2')
+const em = document.querySelector('#profile em')
+// console.log(em)
+image.src = traveler.photo 
+image.alt = traveler.name
+h2.textContent = traveler.name
+em.textContent = traveler.nickname
```

Comment:
Nice! Sylwia would be proud that you remembered the alt attribute!

Would like the variables to be more specific,

```
travNameHeader
travNicknameEm
travPicImg
```